development/sass/top-nav.scss

```diff
@@ -0,0 +1,60 @@
+$nav-fonts: "Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
Note, the underscore beginning the file tells sass it's an include and will not be output as a css file.